tests/cases/fourslash/organizeImports4.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference path="fourslash.ts" />
+
+// Regression test for GH#43107
+
+//// import * as something from "path";/** 
+////  * some comment here
+////  * and there
+////  */
+//// import * as somethingElse from "anotherpath";
+//// import * as AnotherThing from "somepath";/** 
+////  * some comment here
+////  * and there
+////  */
+//// import * as AnotherThingElse from "someotherpath";
+
+
+verify.organizeImports(''); 
```

Comment:
All of these imports are unused so they will just be deleted. We probably do want to test that but you will want to also add tests in which the imports are used, and with non-imports between the imports, also with trailing trivia.

Reply:
Good catch I was so focused on the issue that I didn't even considered. I did and the crash still happens. Working on it.

Reply:
Fixed.